src/test/java/com/googlecode/objectify/test/TransactionTests.java

```diff
@@ -351,8 +351,9 @@ public int getCommitCount() {
 	}
 
 	/**
+	 * @NorbertParrag: 2020.02.12. I removed ConcurrentModificationException retry mechanism, we got to many of them in Aodocs

```

Comment:
Aodocs shouldn't mentioned here.
The useless test should be removed and not just disabled.

Reply:
https://github.com/aliz-ai/objectify/pull/2

Reply:
Btw this is a public repo